docs/configuration.md

```diff
@@ -457,6 +457,8 @@ Launch the synchronization and then exit the application.
 
 #### `synchronization-enabled`
 
+> The option is deprecated and will be removed in a future release. Use "feature.synchronization" instead.
```

Comment:
Use the same deprecated position and formatting as other deprecated options.

Reply:
And that exact sentence should be used in the callback of the `synchronization_enabled` option.